group/lc/main.tf

```diff
@@ -3,13 +3,13 @@
 ## Creates security group
 resource "aws_security_group" "sg_asg" {
   description = "${var.stack_item_fullname} security group"
-  name_prefix = "${var.stack_item_label}-asg-"
+  name_prefix = "${var.stack_item_label}${var.sg_name_suffix}"
```

Comment:
This convention was invented by us. If someone wants to set it to whatever, it's fine by us.